src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -3,22 +3,62 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
+import java.util.Optional;
+
 import seedu.address.model.Model;
+import seedu.address.model.attribute.Attribute;
+
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists all persons in the address book to the user. Attribute can be specified to show only specific attributes.
  */
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "Listed all clients";
+
+    public static final String MESSAGE_SUCCESS_ATTRIBUTE = "Listed all clients and filtered specified attribute";
```

Comment:
Would it be possible for the success message to output the attribute that was specified as well? Like "Listed all clients with address attribute as filter."

Reply:
Great suggestion, done!